rites/roman1969/src/locales/pt-br.ts

```diff
@@ -276,6 +291,8 @@ export const locale: Locale = {
       'São Dionísio, bispo e mártir, patrono da Cidade e da Diocese de Saint-Denis',
     divine_mercy_sunday: '2º Domingo do Tempo Pascal ou Domingo da Misericórdia',
     dominic_de_guzman_priest: 'São Domingos, presbítero',
+    // src: https://www.cnbb.org.br/missal-romano-calendario-proprio-dos-santos-brasil/
+    dulce_lopes_pontes_virgin: 'Santa Dulce Lopes Pontes, virgem',
```

Comment:
@TobiTenno, do we want to require the source reference on the same line where the cited value is located or not? I don’t think we need to, but I want to ask this before we merge it. 🤔

Reply:
In this case, all of them are on the same line except for this one, however, I don’t it is a huge effort to move them above the line. I let you decide, I personally don’t care much, as I want them to be objects anyway, but having interim notes will be very useful when we work on the source objects.